src/components/overview-table.js

```diff
@@ -79,25 +97,54 @@ function FileGroupTableRows({ fileGroup, processFiles, timestamp }) {
     );
 }
 
-function FileGroupTable({ fileGroup, processFiles, timestamp }) {
+function FileGroupTable({
+    fileGroup,
+    processFiles,
+    availableInputs,
+    timestamp,
+}) {
     return (
         <>
             <FileGroupTableHead fileGroup={fileGroup} />
             <FileGroupTableRows
                 processFiles={processFiles}
+                availableInputs={availableInputs}
                 fileGroup={fileGroup}
                 timestamp={timestamp}
             />
         </>
     );
 }
-
-function FileDataRow({ processFile, fileGroup, timestamp }) {
+function FileDataRow({ processFile, availableInputs, fileGroup, timestamp }) {
+    const [open, setOpen] = useState(false);
     let fileType = processFile.fileType;
+    let processFilename = processFile.fileName;
+    const [selectedFilename, setSelectedFilename] = useState('');
     let processFileStatus = processFile.processFileStatus;
     let lastModificationDate = gridcapaFormatDate(
         processFile.lastModificationDate
     );
+    const handleClickOpen = (event) => {
+        setOpen(true);
+        setSelectedFilename(event.target.value);
+    };
+
+    const handleClose = () => {
```

Comment:
Don't you reset the `selectedFilename` value when the Dialog closes?
If you do, you'll need to change the `selectFile` function a little in order to perform the fetch before closing and resetting selected filename.